apis/dex/api.proto

```diff
@@ -1,9 +1,8 @@
 syntax = "proto3";
+option java_package = "com.coreos.dex.api";
```

Comment:
I think this option is irrelevant for us and can be removed entirely.

Reply:
I think it's required by certain proto lint rules. In any case, it should be solved in the upstream repo.

(BTW this file will go away once we tag a new api version in the dex repo. Then we can depend on the api package directly)